File: PYTHON/Cap07/c07ex04.py
# c07ex04.py

def fatorial(n):
  fat = 1
  for i in range(1, n+1):
    fat = fat * i
  print(f"Fatorial = {fat}")

print("Qual fatorial: ", end="")
limite = int(input())

fatorial(limite)

input("\nTecle <Enter> para encerrar... ")

File: KOTLIN/Cap07/c07ex04.kt
// c07ex04.kt

fun fatorial(N: Int) {
    var fat = 1
    for (i in 1..N) {
        fat *= i
    }
    println("Fatorial = $fat")
}

var limite: Int = 0

fun main() {
    print("Qual fatorial: ")
    limite = readLine()?.toIntOrNull() ?: 0

    fatorial(limite)

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
